app.rb

```diff
@@ -0,0 +1,132 @@
+require './book'
+require './classroom'
+require './teacher'
+require './student'
+require './person'
+require './rental'
+
+class App
+  attr_accessor :people, :rentals, :books
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts '#=> No books are available yet' if books.empty?
+    books.each do |book|
+      puts "#=> Title: #{book.title} Author: #{book.author}"
+    end
+    puts people.length
+  end
+
+  def list_all_people
+    puts '#=> No person is available yet' if people.empty?
+    people.each do |person|
+      puts "#=> [#{person.class}] ID: #{person.id} Name: #{person.name} Age: #{person.age}"
+    end
+    puts people.length
+  end
+
+  def create_teacher
+    print "Teacher's specialization: "
+    specialization = gets.chomp
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    teacher = Teacher.new(age, specialization, name)
+    people << teacher
+    puts ''
+    puts '#=> Teacher created successfully'
+  end
+
+  def create_student
+    print 'Age: '
+    age = Integer(gets.chomp)
+    print 'Name: '
+    name = gets.chomp
+    print 'Parent permission? [y/n]: '
+    parent_permission = gets.chomp.downcase
+
+    case parent_permission
+    when 'n'
+      people << Student.new(nil, age, name, parent_permission: false)
+    when 'y'
+      people << Student.new(nil, age, name, parent_permission: true)
+    else
+      '#=> Invalid option, please try again'
+    end
+
+    puts ''
+    puts '#=> Student created successfully'
+  end
```

Comment:
- [ ] Please fix your student and teacher class arguments because when I create any student or teacher it gives me an error and the error is `wrong number of arguments (given 0, expected 1)`  so kindly fix this error as well :slightly_smiling_face: 

**Teacher**
![Screenshot from 2023-08-31 16-48-54](https://github.com/aima98/OOP-School-Library1/assets/45226848/ea219bf0-af4c-470e-b8d8-4dd8333d9550)

**Student**
![Screenshot from 2023-08-31 16-48-09](https://github.com/aima98/OOP-School-Library1/assets/45226848/0542c7e8-91f5-466d-b762-f9579a0a1562)